components/installer/pkg/components/server/configmap.go

```diff
@@ -45,7 +45,7 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 			MaxAgeMs: 259200000,
 			Secret:   "Important!Really-Change-This-Key!", // todo(sje): how best to do this?
 		},
-		DefinitelyGpDisabled: false,
+		DefinitelyGpDisabled: ctx.Config.DisableDefinitelyGP,
```

Comment:
> Bonus points for checking that `definitelyGpDisabled` in `gitpod.yaml` is set to `false` when `disableDefinitelyGp` is not set in `gitpod.config.yaml` file.

`false` is the default value for bool values in Go. When the value is not given, `ctx.Config.DisableDefinitelyGP` is `false`.

The more interesting question is what happens if you don't specify a bool value in your config. However, since the type is bool providing something like `disableDefinitelyGp: fake` would lead to:
```
Error: error loading config: error unmarshaling JSON: while decoding JSON: json: cannot unmarshal string into Go struct field Config.disableDefinitelyGp of type bool
```
Is this good enough?

Reply:
Yup, I'm happy with that.

It's probably worth setting the tags as `json:"disableDefinitelyGp,omitempty" validate:"boolean"` to give a bit of protection to avoid someone putting in `fake` etc